### PR TITLE
Release 6.4.0

### PR DESCRIPTION
**Pull request by TheMeier**
update version

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-systemd",
-  "version": "6.3.1-rc0",
+  "version": "6.3.1",
```
Comment (bastelfreak) [inline metadata.json:3]: There are also some enhancements listed in the changelog, so it has to be a minor release. Please update it to 6.4.0 and regenerate the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
...
+
+**Implemented enhancements:**
+
+- Support `StartLimitIntervalSec` and  `StartLimitBurst` [\#412](https://github.com/voxpupuli/puppet-systemd/pull/412) ([traylenator](https://github.com/traylenator))
+- unit type: add `ConditionPathIsMountPoint` [\#408](https://github.com/voxpupuli/puppet-systemd/pull/408) ([fragfutter](https://github.com/fragfutter))
+- Allow percent \(%\) character in unit names. [\#401](https://github.com/voxpupuli/puppet-systemd/pull/401) ([traylenator](https://github.com/traylenator))
+
+**Fixed bugs:**
+
+- manage\_unit service\_entry Environment can't have multiple values [\#404](https://github.com/voxpupuli/puppet-systemd/issues/404)
```
Comment (bastelfreak) [inline CHANGELOG.md:19]: I added the skip-changelog label to this issue because we have the related PR already in the changelog

**Comment by bastelfreak** [comment]
I added the skip-changelog label to this PR so it  doesn't show up in the next release

**Comment by TheMeier** [comment]
@bastelfreak better now? :)

**Review by bastelfreak — APPROVED** [review]
looks good. do you want to merge & run the release rake task?

**Pull request merged**